rfcs/openapi_generator/pyopenapi/generator.py

```diff
@@ -468,12 +468,14 @@ def _build_operation(self, op: EndpointOperation) -> Operation:
             builder = ContentBuilder(self.schema_builder)
             first = next(iter(op.request_params))
             request_name, request_type = first
+
+            from dataclasses import make_dataclass
+
             if len(op.request_params) == 1 and "Request" in first[1].__name__:
```

Comment:
can you just remove the `if` completely? Then you will get a class named `FooRequestRequest` but that's OK. It is an implementation detail anyway.

Reply:
I tried remove the `if`, but it needs some work to generate the `*Wrapper` class for the `request` class. E.g. https://github.com/meta-llama/llama-stack/commit/838ab91ebf7ea13317f94b0a07e84a271e6b9e20